addressing/README.md

```diff
@@ -0,0 +1,334 @@
+# Addressing in libp2p
+> How network addresses are encoded and used in libp2p
+
+| Lifecycle Stage | Maturity      | Status | Latest Revision |
+|-----------------|---------------|--------|-----------------|
+| 1A              | Recommendation | Active | r0, 2021-07-222  |
+
+
+Authors: [@yusefnapora]
+
+Interest Group: [@mxinden]
```

Comment:
hey we are all interested in this! Mind expanding the group? stebalien, raulk, marten, vyzo, etc.

Reply:
Haha, sure thing @vyzo! I much rather be among many instead of all by myself. Though do expect many GitHub pings to the whole interest group in the future :innocent:

Reply:
```suggestion
Interest Group: [@mxinden, @Stebalien, @raulk, @marten-seemann, @vyzo]
```